docs/UserGuide.md

```diff
@@ -75,26 +75,46 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a volunteer: `vadd` [COMING SOON]
 
-Adds a person to the address book.
+Adds a volunteer to an event.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `vadd vid/VOLUNTEER_ID /to eid/EVENT_ID` or `vadd vn/VOLUNTEER_NAME /to en/EVENT_NAME`
```

Comment:
is there a need for /to here? vid/ and eid/ is sufficient imo since its q intuitive.